code/src/App.js

```diff
@@ -1,12 +1,30 @@
 import React from 'react'
 import data from './data.json'
 
-console.log(data)
+import { Album } from './components/Album'
+import { Header } from './components/Header'
+import { Footer } from './components/Footer'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="site-container">
+      <Header />
+        <div className="container">
+          {data.albums.items.map((item) => {
+            return <Album
+              img = {item.images[1].url}
+              title = {item.name}
+              artist={item.artists.map((item) => 
+                <a className="artistLink" href={item.external_urls.spotify} key={item.name} target="_blank" rel="noopener noreferrer"> 
```

Comment:
One suggestion is to use id instead of name as a key in the future since it can cause confliction if the same artist has more than one release. Like this  `key = {item.id}` .